pkg/download/list.go

```diff
@@ -1,35 +1,150 @@
 package download
 
 import (
+	"context"
+	"errors"
 	"net/http"
 	"strings"
 
+	"github.com/sirupsen/logrus"
+
+	"github.com/gomods/athens/pkg/log"
+
 	"github.com/bketelsen/buffet"
 	"github.com/gobuffalo/buffalo"
 	"github.com/gobuffalo/buffalo/render"
+	"github.com/gomods/athens/pkg/config/env"
 	"github.com/gomods/athens/pkg/paths"
 	"github.com/gomods/athens/pkg/storage"
+	"github.com/google/go-github/github"
+	"github.com/marwan-at-work/vgop/semver"
```

Comment:
@marwan-at-work I remember you saying you had to copy some logic from the go modules tree to verify semvers. Instead of that would you be open to using https://github.com/Masterminds/semver, so you don't have to maintain more code?

Reply:
@arschles i looked into this package and they seem to be relaxed on the `v` prefix restriction (they allow `1.3.2` while semver.IsValid would not allow it - not sure if there are other inconsistencies between the Masterminds/semver and go/semver - should we just use Masterminds/semver anyway?